README.md

```diff
@@ -6,43 +6,43 @@
 This is a library implementing a simple pattern that encourages modularity and
 the Single Responsibility Principle around _doing_ things primarially with ecto.
 
-The goal is a module whose sole responsibility is doing one thing, removing that
-responsibility from the schema (model) module. Some examples might be:
-
-* CreatePost
-* UpdatePost
-* DeletePost
+You can think of this as a second layer of domain modeling that happens in your
+application. You existing schema (model) layer represents the data itself and
+it's relationships. The new interaction layer represents high level actions or
+events that happen on your domain. Some good examples in a blog domain might be:
```

Comment:
You may want to de-emphasize using interactions for simple cases like create/update/delete where little can go wrong and instead show the strength of interactors in handing complex interaction workflows like in only creating a `invoice` record when a `payment_type` is correctly validated, persisted, and checked against a payment provider.

In discussions about interactors I've heard a lot of YAGNI like in this discussion https://news.ycombinator.com/item?id=7335211 .My feeling is that interactors are poor candidates for simple things, but good candidates once your business logic begins to sprawl.